build/launchWebTest.js

```diff
@@ -36,7 +37,7 @@ async function go() {
         await test_web.runTests({
             browserType: 'chromium',
             verbose: true,
-            headless: true, // Set this to false to debug failures
+            headless: isCI ? false : true, // Set this to false to debug failures (true on CI to support capturing screenshots when tests fail).
```

Comment:
Your comment is backwards

Reply:
It's 'false' on CI to support screenshots

Reply:
updated comments.